setup.py

```diff
@@ -67,6 +67,17 @@ def _safe_copy_tree(src, dst, safe=None):
 if "TOXENV" in os.environ and "SETUPPY_CFLAGS" in os.environ:
     os.environ["CFLAGS"] = os.environ["SETUPPY_CFLAGS"]
 
+test_req = [
+    "pre-commit",
+    "pytest",
+    "pytest-cov",
+    "tox",
+    "pytest-remotedata>=0.3.2",
```

Comment:
I had to update pytest as well, so there might be dependency on that too. Not sure how easy that is to test though. Perhaps minimum of > 5.0.0. You could also just move it to the latest version.